package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "password-manager-resources",
```

Comment:
Should probably mark this as private, yeah?
```suggestion
    "name": "password-manager-resources",
    "private": true,
```

Reply:
Sure, up to you guys.